Lib/test/test_email/test_message.py

```diff
@@ -1055,6 +1055,13 @@ def test_get_body_malformed(self):
         # AttributeError: 'str' object has no attribute 'is_attachment'
         m.get_body()
 
+    def test_get_bytes_payload_with_quoted_printable_encoding(self):
+        payload = memoryview(b'Some payload')
```

Comment:
```suggestion
        # We use a memoryview() to avoid directly accessing the private payload
        # and to prevent using the dedicated paths for string or bytes objects.
        payload = memoryview(b'Some payload')
```